config/cmd/cmd.go

```diff
@@ -278,7 +278,7 @@ var (
 			Name:    "service_namespace",
 			EnvVars: []string{"MICRO_NAMESPACE"},
 			Usage:   "Namespace the service is operating in",
-			Value:   "go.micro",
+			Value:   "micro",
```

Comment:
Changing the default. Will this have any adverse impact on people who are already running micro?

Reply:
No, the usage of this flag is still being implemented, right now it's not being used.